pom.xml

```diff
@@ -23,6 +23,8 @@
 		<mapstruct.version>1.5.2.Final</mapstruct.version>
```

Comment:
@vpetrovaa use XCOP, the linter for your XML flies

Reply:
@h1alexbel add it to the pipelines and fix pom.xml using xcop --fix https://github.com/vpetrovaa/elasticsearch-microservice/pull/4/commits/7aa708799a84e0da439103f95c8164a376ad476e

Reply:
@vpetrovaa I suggest using `--fix` only before the code is sent to the repository
when code gets inside the repo, we don't fix mistakes, we fail the build if any mistakes found

WDYT?

Reply:
https://github.com/vpetrovaa/elasticsearch-microservice/pull/4/commits/1aa247b2fc42bfadd7d7a9485197192aab35771a fixed name here